Mention local push support in ha docs for Control4 (#46780)
home-assistant/home-assistant.github.io: source/_integrations/control4.markdown

{% include integrations/config_flow.md %}

## Data updates

This integration is mostly push-based over the local network. State changes are delivered in real time via a WebSocket connection to the Control4 Director. The media player platform is the only exception. It polls a small set of room state (power, volume, mute, source, and
playback status) on a fixed 5-second interval, since Control4 does not push updates for that data.

> During setup or reload of a config entry, the integration contacts the Control4 cloud to obtain a short-lived local token. The token has an expiry date and is refreshed automatically in the background.

Reviewer: ```suggestion
During setup or when the integration reloads, the integration contacts the Control4 cloud to obtain a short-lived local token. The token has an expiry date and is refreshed automatically in the background.
```
